docs/adding-executors.md

```diff
@@ -0,0 +1,48 @@
+---
+permalink: /docs/adding-executors
+---
+
+# Adding Executors
+
+Teaclave supports several function executors currently: `builtin`, `mesapy` and
+`wamr`(WebAssembly Micro Runtime). For more information about current function
+executors, please check [this link]().
+
+However, sometimes service providers may found current executors built in
```

Comment:
In Teaclave, we call "service providers" as "platform providers" who provide the Teaclave platform.